lib/strong.rb

```diff
@@ -0,0 +1,19 @@
+class Str
+
+  def strong_open(input)
+    input.split.each{|x| x.sub!("*", "<strong>") if x[0] == "*"}
+  end
+
+  def strong_close(input)
+    input.split.each {|x| x.sub!("*", "</strong>") if x[-2..-1].nil? != true}
+    #  && x[-2.-1].include?("*")}
+  end
+
+  def strong(input)
+    input.map!{ |x| x.include?("*") ? strong_open(x): x}
```

Comment:
permanent change is ok, but it would be better make a new variable so we understand the flow of data